core/chains/evm/txmgr/txmgr.go

```diff
@@ -604,31 +580,28 @@ func sendTransaction(ctx context.Context, ethClient evmclient.Client, a EthTxAtt
 func sendEmptyTransaction(
 	ctx context.Context,
 	ethClient evmclient.Client,
-	keyStore KeyStore,
+	attemptBuilder txmgrtypes.AttemptBuilder[*evmtypes.Head, gas.EvmFee, common.Address, common.Hash, *assets.Wei, EthTx, EthTxAttempt],
```

Comment:
There are four uses of this long type def. Is it worth naming?
```go
type EthAttemptBuilder = txmgrtypes.AttemptBuilder[*evmtypes.Head, gas.EvmFee, common.Address, common.Hash, *assets.Wei, EthTx, EthTxAttempt]
```

Reply:
I've tried to leave the long type def only in the places where in the future the types will get pulled up a level. imo it's not worth renaming at the moment when the interface could change?